src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.cpp

```diff
@@ -130,7 +130,8 @@ bool HelloTriangleApplication::Initialize()
             DXGI_FORMAT::DXGI_FORMAT_R32G32B32_FLOAT,
             0,
             offsetof(VertexPositionColor, position),
-            D3D11_INPUT_CLASSIFICATION::D3D11_INPUT_PER_VERTEX_DATA, 0
+            D3D11_INPUT_CLASSIFICATION::D3D11_INPUT_PER_VERTEX_DATA,
```

Comment:
Empty line on purpose?

Reply:
yes, should I revert?

Reply:
Yes please, its part of the thing and not something separate? I might be missing something

Reply:
yeah I just though it was weird to have two different fields on the same line, reverting now